pyk4a/config.py

```diff
@@ -8,6 +8,19 @@ class FPS(IntEnum):
     FPS_30 = 2
 
 
+# k4a_image_format_t
+class ImageFormat(IntEnum):
```

Comment:
I would not include this in the PR except if you need it for the calibration

Reply:
will remove